This patch file is a modified version of tiff.js's patches to libtiff, originally available at
https://github.com/seikichi/tiff.js/blob/master/tif_open.c.patch
https://github.com/seikichi/tiff.js/blob/master/tiff.h.patch

Index: tiff-4.0.6/libtiff/tif_open.c
===================================================================
--- tiff-4.0.6.orig/libtiff/tif_open.c
+++ tiff-4.0.6/libtiff/tif_open.c
@@ -98,7 +98,6 @@ TIFFClientOpen(
 	assert(sizeof(int32)==4);
 	assert(sizeof(uint64)==8);
 	assert(sizeof(int64)==8);
-	assert(sizeof(tmsize_t)==sizeof(void*));
 	{
 		union{
 			uint8 a8[2];
Index: tiff-4.0.6/libtiff/tiff.h
===================================================================
--- tiff-4.0.6.orig/libtiff/tiff.h
+++ tiff-4.0.6/libtiff/tiff.h
@@ -28,6 +28,7 @@
 #define	_TIFF_
 
 #include "tiffconf.h"
+#include <stdint.h>
 
 /*
  * Tag Image File Format (TIFF)
@@ -65,17 +66,17 @@
  * strings              unsigned char*
  */
 
-typedef TIFF_INT8_T   int8;
-typedef TIFF_UINT8_T  uint8;
+typedef int8_t   int8;
+typedef uint8_t  uint8;
 
-typedef TIFF_INT16_T  int16;
-typedef TIFF_UINT16_T uint16;
+typedef int16_t  int16;
+typedef uint16_t uint16;
 
-typedef TIFF_INT32_T  int32;
-typedef TIFF_UINT32_T uint32;
+typedef int32_t  int32;
+typedef uint32_t uint32;
 
-typedef TIFF_INT64_T  int64;
-typedef TIFF_UINT64_T uint64;
+typedef int64_t  int64;
+typedef uint64_t uint64;
 
 /*
  * Some types as promoted in a variable argument list
